BlobTierAnalysisTool/BlobTierAnalysisTool/Program.cs

```diff
@@ -22,12 +22,10 @@ class Program
 		private const string ShowContainerLevelStatisticsArgumentName = "/ShowContainerLevelStatistics:";
         private const string AutoChangeTierArgumentName = "/AutoChangeTier:";
 
-		private static Dictionary<StandardBlobTier, Models.StorageCosts> storageCosts = null;
+		private static Dictionary<AccessTier, Models.StorageCosts> storageCosts = null;
 		private static IEnumerable<string> sourcesToScan = null;
 		private static Models.FilterCriteria filterCriteria = null;
 		private static bool showContainerLevelStatistics = false;
-        private static bool autoChangeTier = false;
-        private static bool promptForChangeTier = false;
 
 		static void Main(string[] args)
```

Comment:
It would be great to use async consistently throughout the sample. Currently some methods are async and some are sync.